src/pathfinding_service/model/token_network.py

```diff
@@ -63,6 +65,39 @@ def window(seq: Sequence, n: int = 2) -> Iterable[tuple]:
         yield result
 
 
+def backwards_fee_sender(
```

Comment:
Isn't this the same as in `channel_view.py`? Any reason for the duplication?

Reply:
Nice find, I forgot to delete them when I moved them to the channel view.